pkg/manifest/manifest.go

```diff
@@ -107,10 +109,29 @@ func (e Environments) Names() []string {
 	return maps.Keys(e)
 }
 
+// Account holds all necessary information to access the account API
+type Account struct {
+	// Name is the account-name that is used to resolve user-defined lookup names.
+	Name string
+
+	// AccountUUID is the Dynatrace-account UUID
+	AccountUUID uuid.UUID
+
+	// ApiUrl is the target URL of this account.
+	// It is used when the default account management url is not the target account management url.
+	ApiUrl *url2.URL
```

Comment:
~~Why use the `net/url` type here instead of the `*URLDefinition` that would allow setting in as env var as well?~~

Addressed/changed to URLDefinition in a later commit 👍